train/tr11-200B-ml/tr11-200B-ml.slurm

```diff
@@ -128,7 +128,7 @@ GPT_ARGS=" \
     --checkpoint-activations \
     --abort-on-unmet-fused-kernel-constraints \
     --kill-switch-path $KILL_SWITCH_PATH \
-    --make-vocab-size-divisible-by 1024 \
+    --pad-vocab-size-to 250880 \
```

Comment:
Needs [this PR](https://github.com/bigscience-workshop/Megatron-DeepSpeed/pull/255) to be merged and used